halo2_backend/src/helpers.rs

```diff
@@ -20,7 +20,6 @@ pub enum SerdeFormat {
     RawBytesUnchecked,
 }
 
-// Keep this trait for compatibility with IPA serialization
```

Comment:
If the `CurveRead` trait is simply for IPA compatibility, we can also remove this `CurveRead` trait.
Is it right?